translations/de.log

```diff
@@ -0,0 +1,97 @@
+*****: de
+orig->languageName: 'en', 'de'
+orig->control: 'Control', 'Steuerung'
+orig->print: 'Print', 'Druck'
+orig->status: 'Status', 'Status'
+orig->console: 'Console', 'Konsole'
+orig->setup: 'Setup', 'Setup'
+orig->current: 'Current°C', 'Istwert°C'
+orig->active: 'Active°C', 'Aktiv°C'
+orig->standby: 'Standby°C', 'Standby°C'
+orig->move: 'Move', 'Bewegung'
+orig->extrusion: 'Extrusion', 'Extrusion'
+orig->macro: 'Macro', 'Makro'
+orig->stop: 'STOP', 'STOP'
+orig->extruderPercent: 'Extruder%', 'Extruder%'
+orig->speed: 'Speed ', 'Tempo '
```

Comment:
better translation then "Tempo" is "Geschwindigkeit".
When it is too long, take the english word "Speed"

Reply:
Done.